helm/flowforge/templates/deployment.yaml

```diff
@@ -39,6 +39,7 @@ spec:
         - containerPort: 3000
         securityContext: 
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
```

Comment:
I like this change, but worry it might some day brake. Imaging a developer using `/tmp` to quickly persist something for the scope of a request, there's no indication this doesn't work in production. Right?

Don't know if that's an issue we want to fix, or just document this limitation?

Reply:
We shouldn't be using the filesystem for that sort of thing, it lives in memory or in some external storage (db/redis) if needed then it scales.

Reply:
@hardillb I fully agree! :) But maybe it should be documented that the application runs with a readOnly Filesystem, as opposed to the device-agent in https://flowforge.com/docs/contribute/?